.env.example

```
@@ -31,10 +31,20 @@ OAUTH_KAKAO_LOCAL_REDIRECT_URI=
 # ===============================
 # AWS
 # ===============================
 AWS_S3_ACCESS_KEY=
 AWS_S3_SECRET_KEY=
 AWS_S3_BUCKET_NAME=
 AWS_REGION=
+
+# ===============================
+# Firebase
+# ===============================
+GOOGLE_APPLICATION_CREDENTIALS=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)